bin/format

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python3
```

Comment:
Is `bin/format` the usual spot to put this?

Reply:
I think we can put this anywhere that seems sensible. In this case, since we run it as an executable script I added it here.